test/VirtualAlexaInvoker.test.js

```diff
@@ -266,6 +266,25 @@ describe("virtual alexa runner", () => {
             expect(results[1].interactionResults[0].error).toBeUndefined();
             expect(results[1].interactionResults[0].exited).toBe(true);
         });
+
+        test("Test goto label", async () => {
```

Comment:
please add a case where the assertion fails and the goto doesn't get triggered

Reply:
test added on https://github.com/bespoken/skill-testing-ml/pull/77/commits/41c480d2589971b87b993f1c2bddb2b72d93d4dc